src/alpaca_eval/models_configs/openbuddy-falcon-7b-v6/configs.yaml

```diff
@@ -0,0 +1,13 @@
+openbuddy-falcon-7b-v6:

```

Comment:
I don't see the evaluations for this model, can you remove this configs from the PR or evaluate the model?